frontend/src/features/recommendation/SearchFilter.spec.tsx

```diff
@@ -99,8 +99,7 @@ describe('Filters feature', () => {
     expect(queryAllByTestId(languageFilter, 'autocomplete')).toHaveLength(1);
 
     // Check criteria filters presence
-    // 1 slider per criteria, "Neutral" position by default
-    expect(screen.queryAllByLabelText(/neutral/i)).toHaveLength(3);
+    expect(screen.queryByLabelText('multiple criteria')).not.toBeNull();
```

Comment:
can be simplify with

```suggestion
    expect(screen.queryByLabelText('multiple criteria')).toBeVisible();
```

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/860/commits/486caad8e6d5573cdbc01bc8bf56f8e6579abfb9.